ci/cpu/build.sh

```diff
@@ -68,7 +68,7 @@ pip install git+https://github.com/dask/distributed.git@main
 # BUILD - Package builds
 ################################################################################
 
-gpuci_logger "Build conda pkg for libcudf"
+gpuci_logger "Build conda pkg for dask-cuda"
 gpuci_conda_retry build conda/recipes/dask-cuda --python=${PYTHON}
```

Comment:
It's worth noting that it is important for us to start in the directory relative to `setup.py` as is already being done. Using a different directory doesn't work.

There isn't a good way to start from other directories atm. Please see issue ( https://github.com/conda/conda-build/issues/3754 ) for context.